docs/components/input-controller/gpio.md

```diff
@@ -141,11 +128,11 @@ Click **Create component** and then fill in the attributes for your model:
 
 The following attributes are available for `gpio` input controllers:
 
```

Comment:
Hmmm without text wrapping these are pretty hard to use. I know it's not currently part of the diff but maybe split these out into separate tables or something like with gpio motor? Not sure what the best way to approach this is but if you want to take a stab, this side scrolling isn't ideal.

Reply:
Not going to take this on in this ticket, I agree that this one is cumbersome but the attributes look much worse formatted within the table. I'll make a ticket for this on DOCs